test/integration/start_stop_delete_test.go

```diff
@@ -31,6 +31,7 @@ import (
 func TestStartStop(t *testing.T) {
 
 	runner := NewMinikubeRunner(t)
+	runner.RunCommand("config set WantReportErrorPrompt false", true)
```

Comment:
We should probably set this in our script that runs the e2e tests anyways.  Its only set for some commands.